fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -940,24 +940,18 @@ namespace fileBackup {
 	// Backup and Restore taskFunc definitions will inherit from one of the following classes which
 	// servers to catch and log to the appropriate config any error that execute/finish didn't catch and log.
 	struct RestoreTaskFuncBase : TaskFuncBase {
-		virtual Future<Void> handleError(Database cx, Reference<Task> task, Error const &error) {
-			return RestoreConfig(task).logError(cx, error, format("'%s' on '%s'", error.what(), task->params[Task::reservedTaskParamKeyType].printable().c_str()));
-		}
-		virtual std::string toString(Reference<Task> task)
-		{
-			return "";
-		}
-	};
+	    Future<Void> handleError(Database cx, Reference<Task> task, Error const& error) final override {
+		    return RestoreConfig(task).logError(cx, error, format("'%s' on '%s'", error.what(), task->params[Task::reservedTaskParamKeyType].printable().c_str()));
+	    }
+	    virtual std::string toString(Reference<Task> task) const { return ""; }
+    };
 
 	struct BackupTaskFuncBase : TaskFuncBase {
-		virtual Future<Void> handleError(Database cx, Reference<Task> task, Error const &error) {
-			return BackupConfig(task).logError(cx, error, format("'%s' on '%s'", error.what(), task->params[Task::reservedTaskParamKeyType].printable().c_str()));
-		}
-		virtual std::string toString(Reference<Task> task)
-		{
-			return "";
-		}
-	};
+	    Future<Void> handleError(Database cx, Reference<Task> task, Error const& error) final override {
+		    return BackupConfig(task).logError(cx, error, format("'%s' on '%s'", error.what(), task->params[Task::reservedTaskParamKeyType].printable().c_str()));
+	    }
+	    virtual std::string toString(Reference<Task> task) const { return ""; }
+    };
```

Comment:
Use tabs instead? You formatted this file with spaces.

Reply:
I applied `git clang-format` to everything, I think the difference arising because this file did not already comply with our `.clang-format` file.